challenges/objects-arrays.js

```diff
@@ -59,12 +99,18 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (let i = 0; i < graduates.length; i++) {
+  contactInfo.push(graduates[i].first_name, graduates[i].email)
```

Comment:
you'll wanna use a template literal here (with the back tics and the dollar signs) so we push one thing instead of two. ``` contactInfo.push(`${graduates[i].first_name} ${graduates[i].email}`) ```